### PR TITLE
Add create-application route and GitOpsCluster

### PR DESCRIPTION
**Pull request by chenz4027**
Signed-off-by: Maggie Chen <magchen@redhat.com>

- Added Advanced Configuration tab
- Added terminology section for the Advanced Configuration tab
- Added the route for create-application
- Added the method of getting gitopscluster resource and argo namespace from it

### CODE REVIEW

File: frontend/src/atoms.tsx [hunk 1]
```diff
@@ -243,7 +243,7 @@ export function LoadData(props: { children?: ReactNode }) {
         addSetter(ChannelApiVersion, ChannelKind, setChannelsState)
         addSetter(PlacementRuleApiVersion, PlacementRuleKind, setPlacementRulesState)
         addSetter(SubscriptionApiVersion, SubscriptionKind, setSubscriptionsState)
-        addSetter('apps.open-cluster-management.io/v1alpha1', 'GitOpsCluster', setGitOpsClustersState)
+        addSetter('apps.open-cluster-management.io/v1beta1', 'GitOpsCluster', setGitOpsClustersState)
```
Comment (fxiang1) [inline frontend/src/atoms.tsx:246]: @chenz4027 Can you import the apiVersion and kind from gitops-cluster.ts just like the other resources? This way we only need to update the version in one place when it changes

Comment (chenz4027) [reply frontend/src/atoms.tsx:246]: Sure I'll make the change

**Review by fxiang1 — APPROVED** [review]
LGTM!

**Pull request merged**